src/OpenSage.Game/GameWindow.cs

```diff
@@ -26,7 +27,7 @@ private void RaiseClientSizeChanged()
             ClientSizeChanged?.Invoke(this, EventArgs.Empty);
         }
 
-        public GraphicsDevice GraphicsDevice { get; }
+        public GraphicsDevice GraphicsDevice => _device;
```

Comment:
I think you can leave this property as it is, and use a local variable for `device` in the constructor, which you then assign to this property.